src/views/api-docs-view/index.tsx

```diff
@@ -36,12 +36,12 @@ function ApiDocsView({
 	/**
 	 * We always render the API docs name in a heading-styled element.
 	 *
-	 * When `serviceData` is provided, we'll render the service name
+	 * When `serviceData.name` is provided, we'll render the service name
 	 * in an `h1` element, as the `serviceName` is a more meaningful page title.
 	 * In such cases, our page heading needs to be a `p` element to avoid
 	 * having multiple `h1` elements on the page.
 	 */
-	const pageHeadingTag = serviceData ? 'p' : 'h1'
+	const pageHeadingTag = serviceData && serviceData.name ? 'p' : 'h1'
```

Comment:
```suggestion
	const pageHeadingTag = serviceData?.name ? 'p' : 'h1'
```
💭 Just thinking out-loud, either way is fine.